task/04-arrays-tasks.js

```diff
@@ -85,7 +86,15 @@ function getArrayOfPositives(arr) {
  *    [ 'cat, 'dog', 'raccon' ] => [ 'cat', 'dog', 'racoon' ]
  */
 function getArrayOfStrings(arr) {
-   throw new Error('Not implemented');
+   let result=[]
+   for (let i=0; i<arr.length; i++){
+      if (typeof arr[i]!=="string"){
+         continue;
+      } else {
+         result.push(arr[i])
+      }
+   }
+   return result
```

Comment:
Please, try to solve this using just filter
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter